dkarlovi/symfony-flex-docker/0.1/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "copy-from-package": {
+        ".infra/": ".infra",
+        ".dockerignore": ".dockerignore",
+        "docker-compose.yaml": "docker-compose.yaml"
+    },
+    "env": {
+        "COMPOSE_PROJECT_NAME": "symfony",
```

Comment:
This will also be important when I add image tags in `docker-compose.yaml`, need to know how to name them properly in order to tag images being built. Push and pull also need this.